src/docs/building-a-geospatial-app/0-setup.md

```diff
@@ -1,35 +1,19 @@
 Before we start, let's make sure we have everything necessary to run our app.
 
+* [Setup instructions on Mac and Linux](#/installing-a-coding-environment/installing-tools-mac)
```

Comment:
I believe you can simply use `installing-tools-mac.md` for the path and it will automatically resolve and use the history api, maybe try it out

Reply:
Oh cool. I'll try that.

Reply:
Nope, doesn't work :/